docs/content/glossary.md

```diff
@@ -528,7 +532,7 @@ In the GDK you represent a [SpatialOS entity](#spatialos-entity) as a Unity ECS
 
 ### Unity ECS system
```

Comment:
We capitalise "Entity" because Unity do... but looking again at their ECS docs, they're actually inconsistent 😭 But we might as well keep on capitalising it for now.

As for "system", I've just realised that the S in ECS stands for "system", so we should change this heading to either "Unity ECS" or "Unity Entity Component System". What do you think?

Reply:
oh yeah... just realized that this is kind of awkward...

So the ECS or Entity Component System stands for everything including
- worlds
- entities
- systems :trollface: 

So, the system that we describe here is part of an ECS, so... ECS System is correct, but it does sound a bit off

Reply:
Gaaah... :D OK, in that case I think it's fine as it is - "Unity ECS system". I don't think we need to capitalise "system" as it's just a normal word, rather than A Thing.